client/Pages/EventTypeCreate/Query.elm

```diff
@@ -153,6 +189,7 @@ submitQueryCreate model =
               , Json.object
                     [ ( "name", asString FieldName )
                     , ( "owning_application", asString FieldOwningApplication )
+                    , ( "cleanup_policy", asString FieldCleanupPolicy )
```

Comment:
Shouldn't you also add partition compaction key fields or retention period, when appropriate?

Reply:
Right! Forgot to do that :sweat_smile: